tests/regression/14-osek/17-multicore.c

```diff
@@ -1,4 +1,4 @@
-//SKIP! PARAM: --set ana.activated "['base','threadid','threadflag','escape','fmode', 'OSEK', 'OSEK2', 'stack_trace_set']" --set ana.osek.oil 17-multicore.oil --set ana.osek.taskprefix function_of_ --set ana.osek.isrprefix function_of_
+//SKIP! PARAM: --set ana.activated "['mallocWrapper','base','threadid','threadflag','escape','fmode', 'OSEK', 'OSEK2', 'stack_trace_set']" --set ana.base.privatization protection-old --set ana.osek.oil 17-multicore.oil --set ana.osek.taskprefix function_of_ --set ana.osek.isrprefix function_of_
```

Comment:
`Maingoblint.check_arguments` should already set `protection-old` here. In other OSEK tests we don't have that in `PARAM` either.

Reply:
We produce warnings here, but it was not clear to me from the warning that that actually changed protection.